/**:
    ros__parameters:

        # These are for the RobotLibrary::Control class(es):
        cartesian_damping: [5.0, 0.0, 0.0, 0.0, 0.0, 0.0,
                            0.0, 5.0, 0.0, 0.0, 0.0, 0.0,
                            0.0, 0.0, 5.0, 0.0, 0.0, 0.0,
                            0.0, 0.0, 0.0, 1.0, 0.0, 0.0,
                            0.0, 0.0, 0.0, 0.0, 1.0, 0.0,
                            0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

        cartesian_stiffness: [50.0,  0.0,  0.0, 0.0, 0.0, 0.0,
                               0.0, 50.0,  0.0, 0.0, 0.0, 0.0,
                               0.0,  0.0, 50.0, 0.0, 0.0, 0.0,
                               0.0,  0.0,  0.0, 5.0, 0.0, 0.0,
                               0.0,  0.0,  0.0, 0.0, 5.0, 0.0,
                               0.0,  0.0,  0.0, 0.0, 0.0, 5.0]
        frequency: 500.0
        joint_position_gain: 100.0
        joint_velocity_gain:  10.0
        manipulability_threshold: 0.001
        max_joint_acceleration: 10.0

        # This is for the underlying QP solver class:
        initial_barrier_scalar: 500.0
        barrier_reduction_rate: 0.001
        step_size_tolerance: 0.1
        max_steps: 10
